test/unit/lifecycle/properties.js

```diff
@@ -218,4 +218,36 @@ describe('lifecycle/properties', function () {
     el.prop1 = '';
     expect(el.getAttribute('prop1')).to.equal('');
   });
+
+  describe('270', function () {
```

Comment:
The sub tests will be appended to that, that's just to classify it. Could do that tho.